scripts/rageshake.go

```diff
@@ -93,12 +94,12 @@ func main() {
 		summary := fmt.Sprintf(
 			"%s\n\nNumber of logs: %d\nVersion: %s\nUser-Agent: %s\n", p.Text, len(p.Logs), p.Version, p.UserAgent,
 		)
-		if err := gzipAndSave([]byte(summary), prefix+".log.gz"); err != nil {
+		if err := gzipAndSave([]byte(summary), prefix, "details.log.gz"); err != nil {
 			respond(500, w)
 			return
 		}
 		for i, log := range p.Logs {
-			if err := gzipAndSave([]byte(log.Lines), fmt.Sprintf("%s-%d.log.gz", prefix, i)); err != nil {
+			if err := gzipAndSave([]byte(log.Lines), prefix, fmt.Sprintf("logs-%d.log.gz", i)); err != nil {
```

Comment:
What's the reason for saving each one as a separate file rather than concatenating them all together (perhaps even with a separator)? Do the logs have timestamps?